redirects-transition.conf

```diff
@@ -242,6 +242,10 @@ rewrite ^/info/tips/(.*) https://www.fec.gov/updates/?update_type=tips-for-treas
 rewrite ^/pdf/nprm/(.*) https://www.fec.gov/resources/legal-resources/rulemakings/nprm/$1 redirect;
 rewrite ^/agenda/([0-9]+)/documents/(.*) https://www.fec.gov/resources/updates/agendas/$1/$2 redirect;
 rewrite ^/agenda/([0-9]+)/(.*).pdf https://www.fec.gov/resources/updates/agendas/$1/$2.pdf redirect;
+rewrite ^/agenda/agendas2003/(.*).pdf https://www.fec.gov/resources/updates/agendas/2003/$1 redirect;
+rewrite ^/agenda/agendas2002/(.*).pdf https://www.fec.gov/resources/updates/agendas/2002/$1 redirect;
+rewrite ^/agenda/agendas2001/(.*).pdf https://www.fec.gov/resources/updates/agendas/2001/$1 redirect;
+rewrite ^/agenda/agendas2000/(.*).pdf https://www.fec.gov/resources/updates/agendas/2000/$1 redirect;
```

Comment:
Same changes need to be made as specified here: https://github.com/fecgov/fec-proxy/pull/148#pullrequestreview-300863290

Reply:
Done. Thank you again!